package.json

```diff
@@ -7,6 +7,10 @@
     "packages/*",
     "tools/**/*"
   ],
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0"
```

Comment:
This is probably a stupid question, but since we use yarn instead of npm, what is the reason to include this here?

Reply:
Not a stupid question, yarn is installed via npm, so this impacts yarn (I tested it because I was confused at first), the screenshot above is running a yarn command in the console.